etc/picongpu/pizdaint-cscs/picongpu.profile.example

```diff
@@ -26,12 +42,14 @@ module load cray-hdf5-parallel/1.10.0
 #
 # needs to be compiled by the user
 export BOOST_ROOT=$SCRATCH/lib/boost-1.62.0
-export PNGWRITER_ROOT=$SCRATCH/lib/pngwriter
+export PNGWRITER_ROOT=$SCRATCH/lib/pngwriter-0.6.0
```

Comment:
QUESTION:
If this dependency is compiled by the user anyway why bother to add a version number?

Reply:
just a best practice one can do, so one knows at a glance what version is residing in there